README.md

```diff
@@ -1,92 +1,174 @@
+[![TravisCI Badge][9]][10]
+[![AppVeyor Badge][11]][12]
+[![Coveralls Badge][13]][14]
 
-[![Build Status](https://travis-ci.org/OpenMDAO/OpenMDAO.svg?branch=master)](https://travis-ci.org/OpenMDAO/OpenMDAO)   [![Build status](https://ci.appveyor.com/api/projects/status/33kct0irhbgcg8m1?svg=true
-)](https://ci.appveyor.com/project/OpenMDAO/blue/branch/master)  [![Coverage Status](https://coveralls.io/repos/github/OpenMDAO/OpenMDAO/badge.svg?branch=master)](https://coveralls.io/github/OpenMDAO/OpenMDAO?branch=master)
+# [OpenMDAO 2][0]
+*This version of **OpenMDAO** is in Development Status **[BETA][15]**.*
 
+## Documentation
+Documentation for the latest version can be found [here][2].
 
+Documentation archives for prior versions can be found [here][3].
 
+## Important Notice
+While the API is MOSTLY stable, the **OpenMDAO** development team reserves the
+right to update it and other elements of the code as needed.
 
-OpenMDAO Naming/Legacy OpenMDAO
--------------------------------
+The team will be making frequent updates to the source code, so users are
+encouraged to regularly pull for updates.
 
-PLEASE NOTE: Until recently, this repository was named OpenMDAO/blue. If you had cloned that repository, please update
-your repository name and remotes to reflect these changes.
+### OpenMDAO Versions
+> **PLEASE NOTE**: Until recently, this repository was named **OpenMDAO/blue**. 
+If you had cloned that repository, please update your repository name and
+remotes to reflect these changes. You can find instructions [here][8].
 
-The OpenMDAO 1.7.4 codebase repo has been renamed to OpenMDAO1, and it resides
-at https://github.com/OpenMDAO/OpenMDAO1
+The **OpenMDAO 2.x.y** code has taken the name **OpenMDAO**, and is maintained
+[here][4]. To install the latest release, run `pip install --update openmdao`.
 
-The OpenMDAO 2.x.y code has taken the name OpenMDAO,
-and it resides at https://github.com/OpenMDAO/OpenMDAO.
+The **OpenMDAO 1.7.4** code repository is now named **OpenMDAO1**, and has moved
+[here][5]. To install it, run: `pip install "openmdao<2"` (the quotes around 
+`openmdao<2` are required). 
 
-Installation of 2.x.y code will now work with `pip install openmdao`.
-Installation of 1.7.4 code will now only work with a version specifier: `pip install openmdao==1.7.4`
+The legacy **OpenMDAO v0.x** (versions 0.13.0 and older) of the 
+**OpenMDAO-Framework** are [here][6].
 
-To use the OpenMDAO v0.x legacy version
- (versions 0.13.0 and older) of the OpenMDAO-Framework, go here:
-https://github.com/OpenMDAO/OpenMDAO-Framework
+### Missing Features from OpenMDAO 1.7.4
+Be aware that this new version of **OpenMDAO** is in Development Status **BETA**
+and not all the features of 1.7.4 are currently available.
 
+Here is a list of things that have not yet been developed in 2.x:
 
-OpenMDAO 2
---------------
+* Pass-by-object variables
+* Automatic ordering of groups/components based on data connections
+* Parallel Finite Difference
```

Comment:
Parallel finite difference can be removed from this list.

Reply:
done